src/pages/Checkout/CheckoutPage.vue

```diff
@@ -683,9 +682,16 @@ export default {
 		if (this.showCheckoutStickyExperiment) {
 			fireHotJarEvent('checkout_sticky_experiment');
 		}
+
+		this.determineIfMobile();
+		window.addEventListener('resize', this.throttledResize);
```

Comment:
Actually we probably don't need these event listeners at all. We really only care if the user loads the checkout page in a specific viewport. Mounted won't run when the viewport changes anyways. Why don't we just use this code and simplify these changes:

```
		if (window?.innerWidth) {
			this.initializeCheckoutStickyExperiment();
		}
```